src/main/java/validation/ModelDeclarationValidity.java

```diff
@@ -0,0 +1,97 @@
+package validation;
+
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+/**
+ * Check Keyword rule for Model
+ */
+public final class ModelDeclarationValidity implements Validation {
+
+    public ModelDeclarationValidity(final String code) {
+        this.code = code;
+    }
+
+    @Override
+    public boolean valid() {
+        final String[] lines = this.code.split("\n");
+        final int codeLength = lines.length;
+        for (int currentLine = 0; currentLine < codeLength; currentLine++) {
+            if (!this.lineHasModel(lines[currentLine])) {
+                continue;
+            }
+            if (!this.nothingAfterModel(lines[currentLine])) {
+                this.errLine = currentLine + 1;
+                this.errMsg = "No character allowed after the \"model\" keyword.";
+                return false;
+            }
+            if (this.onlyHasBlankBeforeModel(lines[currentLine])) {
+                this.errLine = currentLine + 1;
+                this.errMsg = "The model name is required.";
+                return false;
+            }
+            if (!this.oneSpacebarBeforeModel(lines[currentLine])) {
+                this.errLine = currentLine + 1;
+                this.errMsg = "There should be only one space between name and \"model\" keyword.";
+                return false;
+            }
+            if (!this.firstCharacterIsCaptial(lines[currentLine])) {
+                this.errLine = currentLine + 1;
+                this.errMsg = "Should be first character is capitalized with model keyword.";
```

Comment:
Should be first character is capitalized with model keyword.
-> The first letter of model name should be capitalized.